pkg/core/native/native_neo.go

```diff
@@ -200,6 +202,13 @@ func newNEO(cfg config.ProtocolConfiguration) *NEO {
 	md = newMethodAndPrice(n.unregisterCandidate, 1<<16, callflag.States)
 	n.AddMethod(md, desc)
 
+	desc = newDescriptor("onNEP17Payment", smartcontract.VoidType,
```

Comment:
BTW, one important thing to me is that we forgot to update Neo's list of supported standards. It should include NEP-27, ditto for C# likely.